http/src/main/resources/jupyter/init-actions.sh

```diff
@@ -177,6 +180,18 @@ END
  read_from_head true
  tag jupyter
 </source>
+END
+
+    # Add stack driver configuration for user start up script
+    tee /etc/google-fluentd/config.d/userStartupScirpt.conf << END
```

Comment:
Rob added similar config for this in a PR just merged....so we can remove this now.